ecosystem/sep-0009.md

```diff
@@ -30,39 +37,66 @@ occupation | 3 | [ISCO08](https://en.wikipedia.org/wiki/International_Standard_C
+'KYC0' | `mobile_number` | phone number | Mobile phone number with country code, in E.164 format
+'KYC0' | `email_address` | string | Email address
+'KYC1' | `birth_date` | date | Date of birth, e.g. `1976-07-04`
+'KYC2' | `birth_place` | date | Place of birth (city, state, country; as on passport)
+'KYC2' | `birth_country_code` | country code | ISO Code of country of birth
+'KYC2' | `bank_account_number` | string | Number identifying bank account
+'KYC3' | `bank_number` | string | Number identifying bank in national banking system (routing number in US)
+'KYC3' | `bank_phone_number` | string | Phone number with country code for bank
+'KYC3' | `tax_id` | string | Tax identifier of user in their country (social security number in US)
+'KYC3' | `tax_id_name` | string | Name of the tax ID (`SSN` or `ITIN` in the US)
+'KYC3' | `occupation` | number | Occupation ISCO code
+'KYC3' | `employer_name` | string | Name of employer
+'KYC3' | `employer_address` | string | Address of employer
+'KYC2' | `language_code` | language code | primary language
+'KYC1' | `id_type` | string | `passport`, `drivers_license`, `id_card`, etc...
+'KYC1' | `id_country_code` | country code | country issuing passport or photo ID as ISO 3166-1 alpha-3 code
+'KYC1' | `id_issue_date` | date | ID issue date
+'KYC1' | `id_expiration_date` | date | ID expiration date
+'KYC1' | `id_number` | string | Passport or ID number
+'KYC1' | `photo_id_front` | binary | Image of front of user's photo ID or passport
+'KYC1' | `photo_id_back` | binary | Image of back of user's photo ID or passport
+'KYC3' | `notary_approval_of_photo_id` | binary | Image of notary's approval of photo ID or passport
+'KYC3' | `ip_address` | string | IP address of customer's computer
+'KYC2' | 'photo_proof_residence' | binary | Image of a utility bill, bank statement with the user's name and address
+
+Legal entity KYC fields
+
+KYC Level | Name | Type | Description
+-----------|-----|------|------------
+'KYC0' | `company_name' | string | Full company name as on the company incorporation papers
+'KYC0' | `company_VAT_number' | string | Company VAT number 
+'KYC0' | `company_registration_number' | string | Company registration number
+'KYC0' | `company_registered_address' | string | Company registered address
+'KYC1' | `number_of_shareholders' | number | Company shareholder number
+'KYC1' | `shareholder_name' | string | Can be a company or a person and should be queried recursively up to the ultimate beneficial owners (with KYC information for natural persons such as above)
+'KYC0' | `image_incorporation_doc' | string | Image of company incorporation documents
+'KYC1' | `image_proof_adress' | string | Image of a utility bill, bank statement with the company's name and address
+'KYC0' | `address_country_code` | country code | country code for current address
+'KYC0' | `state_or_province` | string | name of state/province/region/prefecture
+'KYC0' | `city` | string | name of city/town
```

Comment:
I wonder if we should name-space entity fields to disambiguate. So something like `entity.city` instead of just `city`

Reply:
That would indeed bring more clarity. Maybe we could call it company instead of entity (less abstract).
Your comment also makes me thinking if would be possible to have a kind of hierarchical organization of data, since in fact a company can have as shareholders other companies or persons, and during due diligence process, the financial institutions have to go recursively down the tree until the ultimate beneficial owners (which must be persons). So basically always a company KYC data must be linked to at least a person KYC data (and in many cases to several persons and companies). 
So for a business KYC it might be the case the after first request, the party who's interrogating the other party will send subsequent requests related to the first one. 
I don't want to overcomplicate things here i am just stating how this process is actually taking place in financial institutions. 
If you think it makes sense to explore this side we can go in details there too.

Reply:
I love the idea of prefixing all of these with a dot-notation namespace. I'd vote for `organization`, in keeping with [schema.org](https://schema.org/Organization).

I'd also hold off on more complex setups just yet, let's wait for the need to develop and then we'll have more info on how exactly to solve it.